### PR TITLE
added .opus as supported audio extension

### PR DESCRIPTION
**Pull request by Airop**
added .opus as supported audio extension

### CODE REVIEW

File: libs/libcommon/src/libcommon/viewer_utils/asset.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from libcommon.storage import StrPath, remove_dir
 from libcommon.storage_client import StorageClient
 
-SUPPORTED_AUDIO_EXTENSION_TO_MEDIA_TYPE = {".wav": "audio/wav", ".mp3": "audio/mpeg"}
+SUPPORTED_AUDIO_EXTENSION_TO_MEDIA_TYPE = {".wav": "audio/wav", ".mp3": "audio/mpeg", ".opus": "audio/ogg"}
```
Comment (severo) [inline libs/libcommon/src/libcommon/viewer_utils/asset.py:17]:
```suggestion
SUPPORTED_AUDIO_EXTENSION_TO_MEDIA_TYPE = {".wav": "audio/wav", ".mp3": "audio/mpeg", ".opus": "audio/opus"}
```

**Review by severo — COMMENTED** [review]
We also need new unit tests. I'll push them directly, as it's not trivial

**Review by lhoestq — APPROVED** [review]

**Comment by severo** [comment]
I think we don't need to recompute the existing datasets with Opus files (it would remove the converted .wav files, and replace them with the original .opus ones). Let's wait for all of them to be naturally replaced.

**Comment by severo** [comment]
Congrats @Airop for the PR!!!

**Comment by severo** [comment]
also: about the e2e tests: I think they are failing because the PR is done from a branch in an external repo.

**Comment by Airop** [comment]
> Congrats @Airop for the PR!!!

Thank you! :)

**Pull request merged**